src/PanelTraits/Show.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+namespace Backpack\CRUD\PanelTraits;
+
+trait Show
+{
+    /*
+    |--------------------------------------------------------------------------
+    |                                   SHOW
+    |--------------------------------------------------------------------------
+    */
+
+    /**
+     * Add field to the show view
+     *
+     * @param $field
+     * @return $this
+     */
+    public function addShowField($field)
+    {
+        // if the field_definition_array array is a string, it means the programmer was lazy and has only passed the name
+        // set some default values, so the field will still work
+        if (is_string($field)) {
+            $complete_field_array['name'] = $field;
+        } else {
+            $complete_field_array = $field;
+        }
+
+        // if this is a relation type field and no corresponding model was specified, get it from the relation method
+        // defined in the main model
+        if (isset($complete_field_array['entity']) && ! isset($complete_field_array['model'])) {
+            $complete_field_array['model'] = $this->getRelationModel($complete_field_array['entity']);
+        }
+
+        // if the label is missing, we should set it
+        if (! isset($complete_field_array['label'])) {
+            $complete_field_array['label'] = ucfirst($complete_field_array['name']);
```

Comment:
There's a running "issue" here where simply `ucfirst` may not be what people want (I've suggested a callback with a default in the past).